### PR TITLE
Update API Reference Links In Index File 

### PR DESCRIPTION
**Pull request by uvmetal**
Updated per snowy's request. This PR fixes the API doc reference links in the index for the browser demo jsfiddle code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,5 +41,8 @@
   "private": true,
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "@cityofzion/neon-js": "^4.6.0"
```
Comment (snowypowers) [inline package.json]: we cannot depend on ourselves

File: examples/browser/README.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ getrawmempool()
 
 Gets a list of unconfirmed transactions in memory.
 
-JSFiddle: [getrawmempool](https://jsfiddle.net/gh/get/jquery/3.0/cityofzion/neon-js/tree/master/examples/browser/getrawmempool/demo)
+JSFiddle: [getrawmempool](https://jsfiddle.net/gh/get/jquery/3.0/phetter/neon-js/tree/master/examples/browser/getrawmempool/demo)
```
Comment (snowypowers) [inline examples/browser/README.md]: why do you want to revert it back to relying on `phetter` namespace as compared to `cityofzion` namespace?

Comment (uvmetal) [reply examples/browser/README.md]: Thanks for catching this. I will resolve. That was from the old test code.

**Comment by uvmetal** [comment]
Closed to fix in another branch.

**Pull request closed without merging**